common/lib/common-utils/src/bufferBrowser.ts

```diff
@@ -4,6 +4,13 @@
  */
 
 import * as base64js from "base64-js";
+/**
+ * Convert base64 or utf8 string to array buffer
+ */
+export function toBuffer(input: string, encoding: string): ArrayBufferLike {
```

Comment:
I like encapsulation here (hiding presence of IsoBuffer), but I wonder if we can find a bit less generic / more concrete name for it? That said, I do not have good suggestions :)

Reply:
you have bufferToString(), naming should be consistent with this API (i.e. it asks it to be stringToBuffer)